main.go

```diff
@@ -63,6 +63,28 @@ func get_random_values(level string) (float32, float32){
     return r1, r2
 }
 
+func get_random_operator() (string){
+    seed := rand.NewSource(time.Now().UnixNano())
+    randgen := rand.New(seed)
+    op := "+-/*"[randgen.Intn(4)]
+    return string(op)
+}
+
+func evaluate_expression(op string, r1 float32, r2 float32) (float32) {
+    switch op {
+    case "+":
+        return r1 + r2
+    case "-":
+        return r1 - r2
+    case "*":
+        return r1 * r2
+    case "/":
+        return r1 / r2
```

Comment:
Ok, so we can have three approaches here:
- pretend it's a feature, catch the error and handle it as "That was a trick question",
- not allow `/` whenever `r2 ==0`,
- redraw `r2`, when `r2 == 0` and `/`.

I'll go for option 3 for now, but we should keep the other ones in mind just in case.